components/coding/GoalsSectionComponent.test.ts

```diff
@@ -0,0 +1,64 @@
+import { returnGoalStyle } from "./GoalsSectionComponent";
+
+test("Test returnGoalStyle function - highlighted red", async () => {
+    //Arrange
+    const goal =
+        {"__typename":"user_goals",
+        "createdAt": new Date(2022-11-29),
+        "goalName":"work with Raveen on goals",
+        "goalNotes":null,
+        "id":"a1b08c36-2c8a-49e5-b9c4-f653f826fcee",
+        "updatedAt": new Date(2022-11-29),
+        "userId":"BR9u18SJzvVNzcrkEIcJDPxv1ws1",
```

Comment:
Is this your real user id? We shouldn't include this in our code for security reasons. I'm pretty sure your test would work with a dummy user id right?